src/template/template.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from "@nestjs/common";
+import { readFileSync } from "fs";
+import { TemplateContent } from "./dto/template-content.dto";
+import * as Handlebars from "handlebars";
+import mjml2html = require('mjml');
+
+@Injectable()
+export class TemplateService {
+  public getResetPasswordEmailTemplate(locale: string): TemplateContent {
```

Comment:
Let’s do the mjml to html conversion here since it depends on the template file content. These type of methods should always return html even the source file is in mjml format. If the source file is already a html file, we can just read from it.